Weather App/UI/SettingsVC/SettingsCoordinator.swift

```diff
@@ -20,7 +22,54 @@ class SettingsCoordinator: Coordinator {
         let viewController = SettingsViewController()
         let viewModel = SettingsViewModel()
         
-        //TODO: setup viewModel
+        settingsData = AppSettings.loadSettings()
+        viewController.updateView(data: settingsData!)
```

Comment:
Coordinator calls a method on the viewController to update the view - should a coordinator do that? 
Force unwrapped settings data? Is that safe?

Reply:
The coordinator updates the view the first time it's shown (it has to happen somewhere, this seems like the best place).
The settingsData was initialised the line before. AppSettings.loadSettings never returns nil.

Reply:
Settings loading moved to viewModel.
View updating moved to viewController's viewDidLoad method.
Removed any use of optionals.